README.md

```diff
@@ -38,15 +38,26 @@ make workers-generate-cfg
 
 **WARNING: If all workers are already started, you must stop them before regenerate the docker-compose file using `make workers-stop`.**
 
+### EFK stack for monitoring
+
+In order to perform an applicative monitoring (viz. save and access the logs of the docker containers), an EFK (Elasticsearch-Fluentd-Kibana) stack is used. More informations on it [here](ek/README.md).
```

Comment:
Can we replace:
More informations on it here

by something like:
A complete description on it is placed in [monitoring](monitoring/README.md) folder

Reply:
ok